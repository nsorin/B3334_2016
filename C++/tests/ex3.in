R r1 10 10 -10 -10
R r2 15 15 0 0
OR r12 r1 r2
MOVE r12 100 100
HIT r12 -9 -9
HIT r1 -9 -9
EXIT
