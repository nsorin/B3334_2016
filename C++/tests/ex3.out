OK
OK
OK
OK
NO
YES
